modules/apps/frontend-theme/frontend-theme-unstyled/src/main/resources/META-INF/resources/_unstyled/templates/init.ftl

```diff
@@ -230,51 +229,13 @@
 	/>
 </#if>
 
-<#assign the_title = "" />
-
-<#if layout.getHTMLTitle(locale)??>
-	<#assign the_title = layout.getHTMLTitle(locale) />
-</#if>
-
-<#if pageTitle??>
-	<#assign the_title = pageTitle />
-</#if>
-
-<#if pageSubtitle??>
-	<#assign the_title = pageSubtitle + " - " + the_title />
-</#if>
-
-<#if tilesTitle?has_content>
-	<#assign the_title = languageUtil.get(locale, tilesTitle) />
-</#if>
-
-<#if page_group.isLayoutPrototype()>
-	<#assign the_title = page_group.getDescriptiveName(locale) />
-</#if>
-
-<#if !tilesTitle?has_content>
-	<#assign the_title = htmlUtil.escape(the_title) />
-</#if>
-
 <#assign
 	layout_friendly_url = layout.getFriendlyURL()
 
 	portlet_id = paramUtil.getString(request, "p_p_id")
-/>
-
-<#if validator.isNotNull(portlet_id) && layout.isSystem() && !layout.isTypeControlPanel() && stringUtil.equals(layout_friendly_url, "/manage")>
-	<#assign the_title = htmlUtil.escape(portalUtil.getPortletTitle(portlet_id, locale)) />
-</#if>
 
-<#if the_title ?has_content && !stringUtil.equals(company_name, site_name) && !page_group.isLayoutPrototype()>
-	<#assign the_title = the_title + " - " + site_name />
-</#if>
-
-<#if htmlTitle??>
-	<#assign html_title = htmlUtil.escape(htmlTitle) />
-<#else>
-	<#assign html_title = the_title + " - " + company_name />
-</#if>
+	html_title = htmlUtil.escape(htmlTitle)
```

Comment:
The removal of all this logic looks great. That will definitely help to reduce the confusion around how this behavior works.